packages/core/src/compile-module.test.ts

```diff
@@ -329,4 +329,80 @@ describe('compileModule()', function () {
       );
     });
   });
+
+  describe('locale variants', () => {
+    it('valid variants', async () => {
+      const mf = new MessageFormat('*');
+      const mp = {
+        'en-US': {
+          key: 'en-US'
+        },
+        'es-MX': {
+          key: 'en-MX'
+        },
+        'es-ES': {
+          key: 'en-ES'
+        }
+      };
+
+      const m = await getModule(mf, mp);
+
+      expect(m['en-US']['key']()).toEqual('en-US');
+      expect(m['es-MX']['key']()).toEqual('en-MX');
+      expect(m['es-ES']['key']()).toEqual('en-ES');
+    });
+
+    it('valid variants', async () => {
```

Comment:
dupe `it` spec names. may consider making them unique

Reply:
ah, the whole test case is duped :)

Reply:
was just a duplicate spec, fixed now